assets/src/application/ui/input/InlineEditInput/InlineEditHeading.tsx

```diff
@@ -1,11 +1,9 @@
 import React from 'react';
 
-import Editable from './Editable';
-import { HeadingProps } from './types';
+import { EditableWithCustomControls, EditableProps } from '@infraUI/inputs';
 
-const InlineEditHeading: React.FC<HeadingProps> = ({ ellipsis, level = 4, ...rest }) => {
-	const ellipsisOptions = ellipsis ? ellipsis : { rows: 2, expandable: false };
-	return <Editable ellipsis={ellipsisOptions} level={level} {...rest} inputType='heading' />;
+const InlineEditHeading: React.FC<EditableProps> = ({ ...props }) => {
+	return <EditableWithCustomControls {...props} />;
 };
```

Comment:
let's change this to just use the `Editable` component without the extra controls.

For the time being we only use these Editable text components for the entity (date & ticket) names, descriptions, and ticket price, and it would be waaaaaaay better to NOT have to click buttons to trigger the edit and save processes.

As well, this needs to add heading tags considering it is named `InlineEditHeading`!!! 😆 
I checked on the Chakra site, and the `Editable` component there seems to work fine when wrapped with a set of `<h1></h1>` tags.

The other option is to get rid of this component completely and simply add the header tags in the client code which may be the better option in the long run instead of dealing with that extra markup via props.

Reply:
Removed this component, since now it's obsolete.

Reply:
the reason I added those custom controls is to provide user with some cue that the field can be edited inline, otherwise it might seem like a hidden feature